lib/ansible/modules/network/nxos/nxos_file_copy.py

```diff
@@ -88,6 +88,7 @@
 from scp import SCPClient
```

Comment:
This needs to be checked if lib is available.  If HAS_SCP is false, it should be handled gracefully in the main function with an instruction as to how to fix it.

```
try:
    from scp import SCPClient
    HAS_SCP=True
except
    HAS_SCP=False
```

See [here](http://docs.ansible.com/ansible/dev_guide/developing_modules.html#module-checklist)